starknet_py/tests/e2e/fixtures/contracts.py

```diff
@@ -44,8 +44,8 @@ def sierra_minimal_compiled_contract_and_class_hash() -> Tuple[str, int]:
     """
     Returns minimal contract compiled to sierra and its compiled class hash.
     """
-    compiled_contract = read_contract("minimal_contract_compiled.json")
-    compiled_contract_casm = read_contract("minimal_contract_compiled.casm")
+    compiled_contract = read_contract("minimal_contract_compiled.json", v1_v2=True)
```

Comment:
Shouldn't it be in `contracts_v1.py` file since it is compiled to sierra?

Reply:
Probably.
I saw this before, was just too lazy to change it 🐵